ulib/fs/ulibfs.fsproj

```diff
@@ -2,6 +2,7 @@
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
     <OtherFlags>--nowarn:0086 --mlcompatibility --nologo</OtherFlags>
+    <OutputPath>..\..\bin\</OutputPath>
```

Comment:
Let's double check if it doesn't produce some extra garbage in the bin directory now.